Add section on host status and substatus (#658)
theforeman/foreman-documentation: guides/common/modules/con_host-global-status-overview.adoc

[id="host-global-status-overview_{context}"]
= Host Global Status Overview

The global status represents the overall status of a particular host.

> The status can have one of three possible values: *OK*, *Warning* or *Error*.

Reviewer: ```suggestion
The status can have one of three possible values: *OK*, *Warning*, or *Error*.
```

You can find global status on the Hosts Overview page.

> The status displays as a small icon next to host name and has a color that corresponds with the status.

Reviewer: ```suggestion
The status displays a small icon next to host name and has a color that corresponds with the status.
```

Maybe `color that indicates the status`?

Author: Much of a muchness with corresponds with v indicates. I think corresponds with sounds better?

Hovering over the icon renders a tooltip with sub-status information to quickly find out more details.
To view the global status for a host, in the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.

OK::
No errors were reported by any sub-status.
This status is highlighted with the color green.

Warning::
While no error was detected, some sub-status raised a warning.
For example, there are no configuration management reports for the host even though the host is configured to send reports.
It is a good practice to investigate any warnings to ensure that your deployment remains healthy.
This status is highlighted with the color yellow.

Error::
Some sub-status reports a failure.
For example, a run contains some failed resources.
This status is highlighted with the color red.

.Search syntax

> 

Reviewer: ```suggestion
```

If you want to search for hosts according to their status, use the syntax for searching in {Project} that is outlined in the {AdministeringDocURL}chap-Administering-Searching_and_Bookmarking[Searching and Bookmarking] chapter of the _Administering {Project}_ guide, and then build your searches out using the following status-related examples:

To search for hosts that have an *OK* status:

[options="nowrap" subs="+quotes"]
----

> 

Reviewer: ```suggestion
```

global_status = ok
----

> To search for all hosts that deserves some attention:

Reviewer: ```suggestion
To search for all hosts that deserve attention:
```
